content/doc/upgrade-guide/2.7.adoc

```diff
@@ -0,0 +1,83 @@
+---
+layout: simplepage
+title:  Jenkins Upgrade Guide
+section: doc
+---
+
+== Upgrading to Jenkins LTS 2.7.x
+
+Each section covers the upgrade from the previous LTS release, the section on 2.7.1 covers the upgrade from 1.651.3.
+
+=== Upgrading to Jenkins LTS 2.7.4
+
+This release only fixed a regression introduced in 2.7.3. No upgrade notes for this release exist.
+
+=== Upgrading to Jenkins LTS 2.7.3
+
+==== Enforced upgrade of detached plugins
+
+https://issues.jenkins-ci.org/browse/JENKINS-37041[JENKINS-37041]
+
+Jenkins will now ensure that detached plugins, i.e. those whose functionality used to be part of Jenkins core but was moved into a plugin, are always at least the bundled version. A message will be printed to the log informing administrators of this:
+
+ Upgraded detached plugins (and dependencies)
+
+=== Upgrading to Jenkins LTS 2.7.2
```

Comment:
"Security by default" is missing. It causes much fun with External configuration-as code solutions. See https://github.com/oleg-nenashev/jenkins.io/blob/8f8d21aba1c340b21adfe9a83afdd7e3f9ea3ab6/content/blog/2016/2016-08-03-jenkins27x-upgrade.adoc#security-by-default-and-configuration-management-tools

I can add it in a follow-up PR if you prefer such approach